pkg/api/kyverno/v1/policy_types.go

```diff
@@ -209,6 +211,8 @@ type ExcludeResources struct {
 	UserInfo `json:",omitempty" yaml:",omitempty"`
 
 	// ResourceDescription contains information about the resource being created or modified.
+	// Requires at least one resource to be specified when under MatchResources, if not,
+	// then ExcludeResources should be present instead.
```

Comment:
"Requires at least one resource ... " this does not apply to `exclude.resources`.